extension/public/popup.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Anek+Malayalam:wght@100;200;300;400;500;600;700;800&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="popup.css"/>
+    <title>Document</title>
+</head>
+<body>
+    <header>
+        <div class="logo">
+            <a>
```

Comment:
What are we href-ing this to, do we need to note that this shoudl be added down the line?

Reply:
I wrapped the logo in an anchor tag, similar to how a typical web page would have it. I'd recommend href-ing either our repo or website to-be.

Reply:
Let’s link it to our repo, could we do that in this PR?